.buildkite/pipeline.yml

```diff
@@ -90,6 +90,7 @@ steps:
       - test-race-linux-arm64
       - test-linux-arm64
       - test-windows
+    allow_dependency_failure: true
```

Comment:
Without this, the test failure annotation would not run if any test failed, which seems to be the opposite of its purpose.